test/support/protos/google/protobuf/any.pb.ex

```diff
@@ -1,54 +0,0 @@
-defmodule Google.Protobuf.Any do
```

Comment:
With these files removed, you can remove the line `~r/^Google\./,` from the `ignore_modules` section of the root mix.exs

Reply:
Addressed.